### ISSUE TITLE
Bug: `currentTime` of `lastPlayedItem` from `/library` not being honored

### ISSUE THREAD

**Issue by harrisi** [TortugaPower/BookPlayer#1009]
## Environment

 - OS Version: 5.0.10-12
 - App Version: 16.6.1

## Description

The app is using it's internal cache for last played item's current time. However, since this can be updated from a third party, the value obtained from `GET /library` should be used.

## Steps to reproduce the behavior
1. Play file for > 10s in web app
2. Pause playback
3. Open app

**Comment by GianniCarlo** [comment]
good catch @harrisi , hmmmm I think there are two possible things we can do here, but first for context, right now the app when it's opened, it fetches the info for the level of the library that the user is seeing (capping at 1 refresh request per minute per level), and the thing is, the last played info comes only at the root level.

So one option would be to just include the last played info for all the levels. The other option, would be to bring back websockets, and just use it for listening to events regarding the last played info, so users can update on the fly 🤔  (cc @yamilnunez ). The third option would be to check on playback with the server, with the drawback that we either hold playback until the response is back, or we start playback and jump the user to the future timestamp, or we start playback, and prompt the user if they want to jump to the latest synced timestamp 🤔, not a fan of this third option, but it's something we can do

**Comment by GianniCarlo** [comment]
per our conversation on Discord, this one was fixed on #1010 🙌  (live on beta)
